assets/js/registration.js

```diff
@@ -0,0 +1,22 @@
+let timeZone = document.querySelectorAll(".num");
```

Comment:
`num` is not good class name, it's hard to understand what it means

Reply:
fixed